externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/ExecuteStatement.scala

```diff
@@ -41,6 +41,10 @@ class ExecuteStatement(
   private val forceCancel =
     session.sessionManager.getConf.get(KyuubiConf.OPERATION_FORCE_CANCEL)
 
+  private val schedulerPool =
+    spark.conf.getOption(KyuubiConf.OPERATION_SCHEDULER_POOL.key).orElse(
+      session.sessionManager.getConf.get(KyuubiConf.OPERATION_SCHEDULER_POOL))
```

Comment:
should we fallback to spark.scheduler.pool?

Reply:
For ad-hoc use case,  I think kyuubi should be transparent for customer.

Reply:
`spark.scheduler.pool` just a local property that user can not set it even using `SparkThriftServer` so we don not need to respect it at config level.

Reply:
make sense